libs/community/langchain_community/document_loaders/__init__.py

```diff
@@ -474,6 +474,7 @@
     from langchain_community.document_loaders.unstructured import (
         UnstructuredAPIFileIOLoader,
         UnstructuredAPIFileLoader,
+        UnstructuredBaseLoader,
```

Comment:
`UnstructuredBaseLoader` was added to this and other files like it to make it accessible for import as a parent class for the SDK Loaders.

Reply:
Good call. I'll prefix it.

Reply:
@efriis, The tests are failing because I changed this to `_UnstructuredBaseLoader` and that class is used in places I didn't anticipate--it's used in `libs/community/langchain_community/document_loaders/lakefs.py` and `libs/community/langchain_community/document_loaders/s3_file.py` for example. I'm not familiar with these loaders, but should I revert making the BaseLoader private? Happy to undo that or fix all the references so they refer to `_UnstructuredBaseLoader`. Lmk